src/components/SpellingVariantInput.vue

```diff
@@ -11,16 +12,20 @@ interface Props {
 interface WikitMenuItem {
 	label: string;
 	description: string;
+	value: string;
 }
 
 const props = defineProps<Props>();
 
 const languageCodesProvider = useLanguageCodesProvider();
 
-const wbLexemeTermLanguages = languageCodesProvider.getLanguageCodes().map( ( lang ) => ( {
-	label: lang,
-	description: '',
-} ) );
+const wbLexemeTermLanguages = languageCodesProvider.getLanguageCodes().map(
+	( [ code, name ]: [ string, string ] ) => ( {
+		label: `${name} (${code})`,
```

Comment:
This should be the `wikibase-lexeme-lemma-language-option` message #TODO

Reply:
Hah, I didn’t realize we already had it in WikibaseLexeme.git ^^

Reply:
now used in the newest commit